src/makeCalculator.js

```diff
@@ -3,8 +3,52 @@
 /**
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const mathOperate = {
+    result: 0,
+    operate(mathMethods, number) {
+      mathMethods.call(this, number);
+
+      return this;
+    },
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(value) {
+      this.result += value;
+
+      return this;
+    },
+    subtract(value) {
+      this.result -= value;
+
+      return this;
+    },
+    multiply(value, result) {
```

Comment:
[CODE STYLE] - The `multiply` method has an unused parameter `result`. It should be removed to avoid confusion.